utils/mega_data_setup.py

```
@@ -341,33 +341,40 @@ def __set_desired_ilvl_list(self, path_to_data=None):
-                    item, item_names, base_ilvls
+                    item, item_names, base_ilvls, base_required_levels
                 )
                 DESIRED_ILVL_LIST.append(snipe_info)
 
         return DESIRED_ILVL_LIST
 
-    def __set_desired_ilvl(self, ilvl_info, item_names, base_ilvls):
+    def __set_desired_ilvl(
+        self, ilvl_info, item_names, base_ilvls, base_required_levels
+    ):
         if "item_ids" not in ilvl_info.keys():
```

Ruff

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)